packages/contracts-bedrock/src/L1/OptimismPortal.sol

```diff
@@ -83,53 +90,46 @@ contract OptimismPortal is Initializable, ResourceMetering, ISemver {
     /// @param success        Whether the withdrawal transaction was successful.
     event WithdrawalFinalized(bytes32 indexed withdrawalHash, bool success);
 
-    /// @notice Emitted when the pause is triggered.
-    /// @param account Address of the account triggering the pause.
-    event Paused(address account);
-
-    /// @notice Emitted when the pause is lifted.
-    /// @param account Address of the account triggering the unpause.
-    event Unpaused(address account);
-
+    // TODO: move to a new inheritable contract so that this modifier can be reused in the
+    // xDM and bridge
     /// @notice Reverts when paused.
     modifier whenNotPaused() {
-        require(paused == false, "OptimismPortal: paused");
+        require(superchain.paused() == false, "OptimismPortal: paused");
         _;
     }
 
     /// @notice Semantic version.
-    /// @custom:semver 1.9.0
-    string public constant version = "1.9.0";
+    /// @custom:semver 2.0.0
+    string public constant version = "2.0.0";
 
     /// @notice Constructs the OptimismPortal contract.
     constructor() {
         initialize({
             _l2Oracle: L2OutputOracle(address(0)),
-            _guardian: address(0),
             _systemConfig: SystemConfig(address(0)),
-            _paused: true
+            _superchainConfig: SuperchainConfig(address(0))
         });
     }
 
     /// @notice Initializer.
     /// @param _l2Oracle Address of the L2OutputOracle contract.
-    /// @param _guardian Address that can pause withdrawals.
-    /// @param _paused Sets the contract's pausability state.
     /// @param _systemConfig Address of the SystemConfig contract.
+    /// @param _superchainConfig Address of the SuperchainConfig contract.
     function initialize(
         L2OutputOracle _l2Oracle,
-        address _guardian,
         SystemConfig _systemConfig,
-        bool _paused
+        SuperchainConfig _superchainConfig
     )
         public
         reinitializer(2)
     {
         l2Sender = Constants.DEFAULT_L2_SENDER;
         l2Oracle = _l2Oracle;
         systemConfig = _systemConfig;
-        guardian = _guardian;
-        paused = _paused;
+        superchainConfig = _superchainConfig;
```

Comment:
Is the `guardian()(address)` getter still present?

Reply:
'tis.